### DISCUSSION TITLE
GFFファイルへのアノテーションの記載方法

### DISCUSSION THREAD

**Discussion by oogasawa** [nig-sc/nigsc_homepage2#282] [Q&A] [answered]
(sc-helpdeskへの質問を質問者の許可を得て代理で起票しております)

RNA-seq解析関連でご助言いただきたくご連絡いたしました。

現在ヒト細胞内でプラスミドによりGFPを発現させ、バルクでRNA-seqを行いfastqファイルを得ました。
そこでHISAT2にてGFP遺伝子も含めてマッピングを試みたいと考えているのですが、
GFPをアノテーションファイル（gffファイル）にどのように記載するべきか悩んでおります。

ゲノムファイル（faファイル）およびアノテーションファイル（gffファイル）にGFPをどのように記載するのがよいか
ご教示いただけますと幸いです。

**Comment by motthy** [comment, marked as answer]
簡単ですが、以下のようなやりかたでよろしいかと思います。
- genomeのfastaにGFPのfastaを追記
- GFFにGFPについての記載を1行追加（1行だけ一番後ろに入れればいいので普通のテキストエディタでOK、もとのGFFのattributeに従う）

**Reply by motthy** [reply]
GFP塩基配列の塩基数と、お使いのGFFのmRNAの行のサンプルを1行貼り付けていただければ、記載例を追記します（9列目の書き方に合わせます）

**Comment by oogasawa** [comment]
参考資料としては以下の通りとのことです。

- https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/using/tutorial_mr
- https://kazumaxneo.hatenablog.com/entry/2019/09/01/073000
